packages/components/src/components/watermark/watermark.tsx

```diff
@@ -3,10 +3,11 @@ import './watermark.scss';
 
 type TProps = {
     image: CSSProperties['backgroundImage'];
+    opacity: CSSProperties['opacity'];
```

Comment:
```suggestion
    opacity?: CSSProperties['opacity'];
```
@heorhi-deriv Since the component already has an opacity `of 0.16` by default you can make the prop optional.